config/input-resource-verify.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I do not like ant solution, for this case, it is over complication.
We still use ant only because it allow us to run checkstyle base on generated class files (before final jar is created).
As you can see it is ugly by configuration to make that hack.

Your case is simpler, you do not need bleeding edge version of this Check, it very simple and unlikely have a problem, but even it got a problem, it has enough inputs during main validation.

Please backup your solution and try to use validation as by maven plugin - https://github.com/checkstyle/checkstyle/blob/master/pom.xml#L457 , by default maven put to validation only src/main folder, you need to configure it to scan xdoc, site, .ci, config, ...... folders.